packages/opc-timeline/src/opc-timeline.scss

```diff
@@ -0,0 +1,106 @@
+@import '@one-platform/opc-styles';
+
+$LOCAL: timeline;
+
+:host {
+  --opc-timeline--BackgroundColor: #DEDEDE;
+  --opc-timeline--Color--step: #689B7A;
+  --opc-timeline--TextColor: #6C6C6C;
+  --opc-timeline--Color--active: #0aa521;
+  --opc-timeline--Color--after-active: #b6e4bc;
```

Comment:
rename this to --opc-timeline--Color--active--after

Reply:
Changed to 
```css
  --opc-timeline--Color--active--after: #b6e4bc;
```